plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiMetadata.java

```diff
@@ -175,11 +167,14 @@ public ColumnMetadata getColumnMetadata(ConnectorSession session, ConnectorTable
     @Override
     public Optional<Object> getInfo(ConnectorTableHandle table)
     {
-        return ((HudiTableHandle) table).getPartitions()
-                .map(partitions -> new HudiInputInfo(
-                        partitions.stream()
-                                .map(HivePartition::getPartitionId)
-                                .collect(toImmutableList())));
+        HudiTableHandle hudiTable = (HudiTableHandle) table;
+        return Optional.of(ImmutableMap.of(
+                "table",
```

Comment:
Can we define these keys as static constants?

Reply:
The object from getInfo is only used to show in the QueryInfo, which could be retrieved via the REST API (http://coordinator/v1/query). It is useful for Presto/Trino developer to inspect the query execution. 

I would define a data class to refine this implementation.